src/backend/PublicInterfaceImpl.c

```diff
@@ -73,6 +75,17 @@ int BSL_API_DeinitLib(BSL_LibCtx_t *lib)
     return BSL_SUCCESS;
 }
 
+int BSL_LibCtx_GetTlmCounters(const BSL_LibCtx_t *lib, BSL_TlmCounters_t *tlm)
```

Comment:
Minor: For functions that can't really fail, it's okay to return void.

Reply:
If there aren't macros for that, it can just check and `return;`
In the original source there were separate macros for void type, int type, and pointer types.

It also doesn't hurt to leave it return a code.